otter/integration/lib/autoscale.py

```diff
@@ -194,10 +204,10 @@ def wait_for_N_servers(
             Otherwise, an exception is raised.
         """
 
-        def check(state):
-            if state[0] != 200:
+        def check((code, response)):
```

Comment:
Oh. Didn't know that. Sorry @sam-falvo for suggesting this. You should probably change to suggested form.